collectors/prometheus.go

```diff
@@ -94,11 +94,16 @@ func NewPrometheusExporter(cfg *PrometheusConfig, lnd *lndclient.LndServices,
 
 	htlcMonitor := newHtlcMonitor(lnd.Router, errChan)
 
+	chanCollector := NewChannelsCollector(
+		lnd.Client, errChan, quitChan, monitoringCfg,
+	)
+	go func() {
```

Comment:
This goroutine can be removed completely.

Reply:
Removed & force pushed